TOC/README.md

```diff
@@ -1,68 +1,63 @@
 # Technical Oversight Committee (TOC)
 
-## Initial Bootstrap Committee
+## Chair ##
 
-The Initial bootstrap committee had to be replaced by a new bootstrap committee.
-This bootstrap committee should be replaced by an elected Jenkins X TOC in the future.
+Current chair is Mårten Svantesson [@msvticket](https://github.com/msvticket), elected in July 6:th 2022.
```

Comment:
extra `:` between 6 and th? does not render as superscript, if that was the intention 🤔

Reply:
Fixed